src/main/resources/templates/list.html

```diff
@@ -18,15 +18,11 @@
 		<input type="submit" value="新規登録">
 		</form>
 	</div>
-	
-	<div>
-		<h1>ようこそ！テスト自動採点システムへ！</h1>
-		<p>下記よりメニューを選択してください</p>
+	<div th:each = "question : ${questionlist}" th:object = "${question}">
```

Comment:
```suggestion
	<div th:each = "question : ${questionlist}">
```
th:object = "${question}"は不要っぽいです。